java-frontend/src/main/java/org/sonar/java/se/checks/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -47,15 +47,7 @@
     "NoSuchElementException will be thrown when invoking method %s() without verifying Optional parameter.");
 
   private enum OptionalConstraint implements Constraint {
-    PRESENT, NOT_PRESENT;
-
-    @Override
-    public String valueAsString() {
```

Comment:
Instead of removing that code I would add a test, because the default behavior of `valueAsString()` from the `Constraint` interface returns an empty String.